src/main/java/org/folio/circulation/domain/PatronGroupRepository.java

```diff
@@ -47,14 +47,18 @@
       .distinct()
       .collect(Collectors.toList());
 
+    if (groupsToFetch.isEmpty()) {
+      return CompletableFuture.completedFuture(HttpResult.succeeded(multipleRequests));
```

Comment:
Personally, I'd use static imports to shorten this code to something like
```suggestion
      return completedFuture(succeeded(multipleRequests));
```